### PR TITLE
LV-624 Corporate Dome Nightmare Insert Changes

### PR DESCRIPTION
**Pull request by BOBAMAx**
# About the pull request

This PR adds three new goon survivor roles and adds them to the corporate dome nightmare insert:
Medic, Engineer, and Team Lead.

Also, I added the original goon survivor to the weyland_survivor faction along with all the new goon survivor roles, as it never had it for some reason. This meant that all the nightmare roles excluding the CL himself would set off the WY motion detector included in the insert. Now none of them do, with the added side effect of being on the PMC IFF network now, just like the regular goon ERT.

There is now a spawner for:
1 CL
1 TL
1 Medic
1 Engineer
and a bunch of regular goons.

I tried to keep the equipment and skills they start with on par with regular survivor roles minus the engineer, who got the ERT fuel satchel to make him more visually distinct from a regular goon because folded welding goggles are basically invisible compared to a medhud. I'm not too concerned about the NSG the leader has because it's the stripped version with no underbarrel flamethrower and half the maps have NSG spawns that nobody uses anyway because the gun kinda sucks.

The goon leader has similar skills to the ERT version which includes engineering skills.

Spawn priority is:
CL = Medic > TL > Engineer > Standard Goons.

# Explain why it's good for the game

The same rationale for why the solaris PMC insert and the CLF crashed ship insert got engineers, medics, and team leads: survivor gameplay without medics or engineers sucks. Getting IB is now no longer a guaranteed death sentence and the plasteel the nightmare insert includes can now actually be used by someone other than a synthetic survivor. The team lead isn't as important but having a distinct shotcaller is good for organization like with FORECON and the CLF insert. Also, he looks cool.

# Testing Photographs and Procedure
nightmare

Tested by forcing the insert and manually spawning myself as all the new and edited roles with the player panel equipment menu. This is my first PR though so I could have messed something up.

# Changelog
:cl: BOBAMA
add: Added Goon Survivor Engineers, Medics, and Team Leads and put them in the LV-624 Corporate Dome Nightmare Insert.
maptweak: slightly adjusted LV-624 Corporate Dome Nightmare Insert Survivor Spawns.
/:cl:

### CODE REVIEW

File: maps/map_files/LV624/standalone/corporatedome.dmm
Comment (Nanu308) [inline maps/map_files/LV624/standalone/corporatedome.dmm:1]: lgtm map wise

**Review by Nanu308 — COMMENTED** [review]
Mapping itself is fine, need to look through survs for the sec goons

**Comment by BOBAMAx** [comment]
now with new blundirised icons, including a new goon medic one.

<details> 
        <summary>Picture</summary> 

goon_medic_ingame

       
</details>

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Pull request closed without merging**